api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8650,7 +8650,38 @@
 			<column name="drug_non_coded" type="varchar(255)" />
 		</addColumn>
 	</changeSet>
-	
+
+	<changeSet id="20160216-1700" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<or>
+				<dbms type="mysql" />
+				<dbms type="oracle" />
+			</or>
+		</preConditions>
```

Comment:
Please remove the precondition. GenerateUuid can be run on any database as it uses generic SQL.